app/helpers/application_helper.rb

```diff
@@ -71,4 +71,14 @@ def submit_or_return_to(f,return_path)
     render partial: 'shared/submit_or_cancel', locals: {f:f, return_path:return_path}
   end
 
+  def unlocked_user
+    if current_user.locked?
```

Comment:
is this to handle people logged in already?  You could just also wipe out the sessions to force a logout on everyone.

Reply:
Yes, that's the idea.  In general I think it will be a good to have so they can immediately lock someone if they're being a jerk